es6-destructuring/es6-destructuring-quiz-notes.md

```diff
@@ -0,0 +1,75 @@
+# es6-destructuring-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is destructuring, conceptually?
+You're able to pull specific values from arrays or properties from objects and assign them to variables in a way to read the code easier.
+- What is the syntax for `Object` destructuring?
+Define a set of variables with the same names as the properties you want to pull from the object within the curly braces and then assign said object to be destructed next to the equal sign
+- What is the syntax for `Array` destructuring?
+same as object destructuring except you use square brackets and everything is written in a linear way
+- How can you tell the difference between destructuring and creating `Object`/`Array` literals?
+object uses curly braces and array uses square brackets
```

Comment:
This answer could use some more explanation based on the location where we find the [] or {}.

Reply:
I thought this was asking for the difference between object and array so it was my fault for not understanding the question properly